src/Core/SettingsEnums.cpp

```diff
@@ -93,6 +93,26 @@ IMPLEMENT_SETTING_ENUM_WITH_RENAME(DefaultDatabaseEngine, ErrorCodes::BAD_ARGUME
     {{"Ordinary", DefaultDatabaseEngine::Ordinary},
      {"Atomic",   DefaultDatabaseEngine::Atomic}})
 
+IMPLEMENT_SETTING_ENUM_WITH_RENAME(DefaultTableEngine, ErrorCodes::BAD_ARGUMENTS,
+    {{"Non", DefaultTableEngine::None},
+     {"Log", DefaultTableEngine::Log},
+     {"StripeLog", DefaultTableEngine::StripeLog},
+     {"MergeTree", DefaultTableEngine::MergeTree},
+     {"CollapsingMergeTree", DefaultTableEngine::CollapsingMergeTree},
+     {"ReplacingMergeTree", DefaultTableEngine::ReplacingMergeTree},
+     {"AggregatingMergeTree", DefaultTableEngine::AggregatingMergeTree},
+     {"SummingMergeTree", DefaultTableEngine::SummingMergeTree},
+     {"GraphiteMergeTree", DefaultTableEngine::GraphiteMergeTree},
```

Comment:
Currently it's not supported, so it's better to remove these options (it may look from possible values that these engines are supported as default engine).